src/client/driver/native-dialog-tracker/index.js

```diff
@@ -118,30 +115,35 @@ export default class NativeDialogTracker {
         };
     }
 
-    _createGeolocationHandler () {
-        return (successCallback, failCallback) => {
-            const url                       = NativeDialogTracker._getPageUrl();
-            const isFirstGeolocationRequest = !nativeMethods.arraySome
-                .call(this.appearedDialogs, dialog => dialog.type === GEOLOCATION_DIALOG_TYPE && dialog.url === url);
+    _geolocationDialogHandler (successCallback, failCallback) {
```

Comment:
Methods or functions should start with a verb. For example, you can name it in this way: `_handleGeolocationDialog`